mcr_scene_segmentation/ros/src/scene_segmentation_node.cpp

```diff
@@ -202,7 +258,7 @@ void SceneSegmentationNode::segment()
 
         //publish cluster, will be used for object_list_merger
         object_list.objects[i].pointcloud = ros_cloud;
-
+        
```

Comment:
please don't add trailing whitespaces

Reply:
I fixed this already, but did not make new commit yet, waiting for @sthoduka for the new pipeline.